frontend/navigation/MainTabNavigator.js

```diff
@@ -1,20 +1,49 @@
-import React from 'react';
-import { Platform } from 'react-native';
-import { createBottomTabNavigator, createStackNavigator } from 'react-navigation';
-
-import TabBarIcon from '../components/TabBarIcon';
-import AnnouncementScreen from '../screens/AnnouncementScreen';
-import DetailedAnScreen from '../screens/DetailedAnnouncements';
-import HomeScreen from '../screens/HomeScreen';
-import ProductsDetailedScreen from '../screens/ProductsDetailedScreen';
-import ProductsScreen from '../screens/ProductsScreen';
-import SettingsScreen from '../screens/SettingsScreen';
+import React from "react";
```

Comment:
You know... I'm getting the feeling that you should install eslint/prettier.... because we have different formatting rules right now 😂